controller.go

```diff
@@ -1107,6 +1107,8 @@ func (c *controller) NewSandbox(containerID string, options ...SandboxOption) (S
 		sb.config.hostsPath = filepath.Join(c.cfg.Daemon.DataDir, "/network/files/hosts")
 		sb.config.resolvConfPath = filepath.Join(c.cfg.Daemon.DataDir, "/network/files/resolv.conf")
 		sb.id = "ingress_sbox"
+	} else if sb.loadBalancerNID != "" {
+		sb.id = "lb_" + sb.loadBalancerNID
```

Comment:
can we do `lb-<id>` so that is consistent with the `1-` of the overlay?

Reply:
Actually, not easily.   The network namespaces in `/var/run/docker/netns` of the form `1-XXXXXXXXX` are generated within the overlay driver and don't have actual `libnetwork.Sandbox` structures associated with them.  I tried doing `lb-XXXXXXXXX` first, but doing so caused immediate failures when using such a sandbox in a `libnetwork.Sandbox` ID.  I am not sure why yet.   I could change the form to be `XXXXXXX_sbox` perhaps if we want to keep some consistency.  WDYT?

Reply:
I still like more the lb prefix, it's much more explainable than the _sbox postfix